EssentialsGeoIP/src/com/earth2me/essentials/geoip/EssentialsGeoIPPlayerListener.java

```diff
@@ -175,6 +175,13 @@ private void downloadDatabase() {
                 logger.log(Level.SEVERE, tl("geoIpUrlEmpty"));
                 return;
             }
+            String licenseKey = config.getString("database.license-key", "");
+            if (licenseKey == null || licenseKey.isEmpty()) {
+                logger.log(Level.SEVERE, tl("geoIpLicenseMissing"));
+                return;
+            } else {
+                url = url.replace("{LICENSEKEY}", licenseKey);
+            }
```

Comment:
No need else branch, because you put "return" in line 181.

Reply:
Fair enough.